### PR TITLE
[_] feat/workspaces (WIP)

### PR DESCRIPTION
**Pull request by apsantiso**
### Overview
The new workspaces feature allows a user to create a collaborative Workspace on Drive with team management, private and public sharing, and permissions functionalities. Refer to the [doc](https://inxt.atlassian.net/wiki/spaces/T/pages/216432643/B2B+-+v1+Requirements) for more info.

#### Steps needed before merging
- [x] Run the migrations first (Migrations are made in a sequential order. Before merging we need to change every new migration file name)
- [X] Create and deploy the new environment variables
- [X] Merge the PR subcomponents in this order: 
   - ~~Migrations~~ / #353 
   - ~~New models and modules~~ / #355 
   - ~~New repositories~~ / #355 
   - The rest (use cases, controllers and changes over already existent modules)

#### **How to contribute?**
Create a branch from this one and add the needed changes. You can merge it without any approval, but **please wait until someone reviews your code to prevent undesired code or changes!**

#### **New Environment variables**

Please add here the new environment variables you need for this module to work. **Remember not to share real environment variables if they are secrets!**

```
WORKSPACES_USER_INVITATION_EMAIL_ID=d-de1ed6df4a9947129c0bf592c808b58d
WORKSPACES_GUEST_USER_INVITATION_EMAIL_ID=d-41b4608fc94a41bca65aab7ed6ccad15
DRIVE_GATEWAY_PUBLIC_SECRET = LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUE0bmVlL0Y5OC8wTjJhbFNKQ3JnMAp5bzJRRysydzR5SGk3ZXVDT3JYYUhENzFmN0NrWExWdHlxWTFRRWNFVnFuQm5QUnVpR1EvSklzWkJKVXhoQTlOCmdwdUpIbVY0aytnMEorRGcxeS9wd3k4L0lNM0FhTnNtWWp4c0NZQUZBSWhqUDZNZlBsVTNTYWdyekVRNklFU3MKeHBzT1JhRXd3WUZIWm42TU50b0FGbktMb3VlMVprUVpSSlVwNGZSSlMvcGNrdVNSNjZFcjFLMjhYckpieGE5egpCNG9SbFJMb0ExQ2cvTFN6ZEFQc2lVMzlSOWtlamxBKzFOMkxLVWFrNVJ3OWN5TkM0N3lHS3ZicSt2TTNYaUFmCk43Wk1teTdkY09aeXcyZW9idFFUVzVtTmR1WElWOHhWeWUzMkpyY3BuYWVqbDlUMG5TWGJEWE9OV2d6N0lWcmoKNFFJREFRQUIKLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0t

# This needs to be reviewed as it could be probably just needed for MinIO
AVATAR_FORCE_PATH_STYLE =
```

***NOTE: This is a Draft, but the preview environments do not work when the PR is in that state, therefore, we need to have this PR in the status "Ready for Review"***

### CODE REVIEW

File: src/modules/gateway/gateway.controller.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@Controller('gateway')
+export class GatewayController {
+  constructor(private gatewayUseCases: GatewayUseCases) {}
+
+  @Post('/workspaces')
+  @ApiOperation({
+    summary: 'Initiates a workspace',
+  })
+  @ApiOkResponse({ description: 'Returns the workspace created' })
+  @Public()
```
Comment (apsantiso) [inline src/modules/gateway/gateway.controller.ts:18]: This is just to skip the globalGuard(AuthGuard) that tries to verify an user based token.

**Review by sg-gs — CHANGES_REQUESTED** [review]

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-c6cf318d2b9c20cfc0ffb3cb5586737cf422acbd

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-d405cbaf3037800830dc1d1730787f33f2c055bc

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-1f909fa049bd0158330a2240e09c40f708cf5a91

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-c507a2d8fddc240dc8e68e0f39ece20d1514cbde

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-c1a19b8dfbdb5c0d85c8db45f18de3bde1a4e865

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-fb647a12bdd77ae6f76df285bb21ce9f8fc59d1f

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-3edc3cbf3b3874856c0ddf9505ab809840332491

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-5b46950c80119edc624fb90be810bbd266fa13bf

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-6b12e881dc0b8488a879e9747728d177be6f0ae9

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-76835a7d99ce16d241696c1fd8dc67b066aa02a7

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-69edc19ab99b81a6a90438f04c7e7a0f52efcfca

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-c672bc3707778b928b915932e099ad1a459b8ac7

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-735f3e44825740960d2b05ceb2262f13107b08bf

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-e2c2dd574ecf98c030ce63506b429146d8476b04

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-d56566e75c284dd862dbc9d02efcf42858d8518f

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-3eeaab11ba3a0cda46a6216c3bf28dda2e78f7d2

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-02fbdc80e8ef5bdc3428cef47ee875e681d967c1

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-64c4ed5848e40313d46f9516de6e46f602904e5f

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-258e44746e5eca876a730b3a0e6044401a62bd93

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-912abbe3eb2130b47b2bfc53f34841a4854faa0c

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-0d904ee54caf74a979a1e4456b9ed6bcaca07a2b

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-846d9214e6df4f12d2a1616780eac7bcb3d4a955

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-46b75d4f267bcfb8d31dd386df030f27108b43b3

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-33ceda9fcf18603091972a25418ea4b4cbe7ca1b

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-feaff6597ba24c071ffb82741e8c62ac5e0f5020

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-41c1aa71e2536ddf019f3cf1409ff75bc0d51e6d

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-dd6257ed86adb4c23ecba2603fbf48967544df8f

**Comment by inxtci** [comment]
> [!CAUTION]
> Preview failed: Not deployed after 5 minutes check 💥
> **Last status received: Progressing**

**Comment by inxtci** [comment]
> [!CAUTION]
> Preview failed: Not deployed after 5 minutes check 💥
> **Last status received: Progressing**

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-9866b8671acdf06bbdde8024155f8de17dcbfc94

**Comment by apsantiso** [comment]
/update-preview drive-server preview-442-YTNiOTMxOTgyYTZlY2FjM2M0Y2VjMT imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-442-YTNiOTMxOTgyYTZlY2FjM2M0Y2VjMT imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-442-a3b931982a6ecac3c4cec142a80cb6f9d0aef64f imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-442-af72fd9d5589d9a7c7451126f5450835c36ba6c2 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-442-af72fd9d5589d9a7c7451126f5450835c36ba6c2 imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-c335395923a8e8dad2451eb5a09d8396429b2c6f

**Comment by apsantiso** [comment]
/update-preview drive-server preview-442-af72fd9d5589d9a7c7451126f5450835c36ba6c2 imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-c335395923a8e8dad2451eb5a09d8396429b2c6f

**Comment by apsantiso** [comment]
/update-preview drive-server preview-442-af72fd9d5589d9a7c7451126f5450835c36ba6c2 imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-e8c9bc432b103922a9c3be1a7d46dec1f0ed8d34

**Comment by apsantiso** [comment]
/update-preview drive-server preview-442-af72fd9d5589d9a7c7451126f5450835c36ba6c2 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-442-af72fd9d5589d9a7c7451126f5450835c36ba6c2 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-f4e98fad68b03afa671ea125f6d969113305547e imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-5cec27c6107d4b86b3fa3750bd69a902299996fb

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-f4e98fad68b03afa671ea125f6d969113305547e imageSuffix=-dev

**Comment by CandelR** [comment]
/update-preview drive-server preview-443-f4e98fad68b03afa671ea125f6d969113305547e imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-a110c66875e10a11f8800a59c9f11b2deab74e17 imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-ce2ee04030b8466e7ecfdd78abb4eac28bd04f9e

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-a110c66875e10a11f8800a59c9f11b2deab74e17 imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-addaa2b9d7344f6dba540245a9c4c2c67ecaa690

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-a110c66875e10a11f8800a59c9f11b2deab74e17 imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-e6926a711c1ff2bec4c7ef3bc66d1a8d361692e9

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-69717b8659e83c860322741482eed782962a10f8

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-64b754824bc29ecc9cfdf5d3538ef7a67e100b3b

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-4321b202539eaaf396f896c6ec98bc918208c5c6

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-2334a256437d026f23d9963e9a3d44c2164267b7

**Comment by apsantiso** [comment]
/update-preview drive-server-wip preview-287-4321b202539eaaf396f896c6ec98bc918208c5c6 imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-e4853175716714113ca5f5e07e398ebf8f4db99e

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-5f748643dbac25052eac2f4ec75acf31d418c77f

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-4afdf562be38dd8e0637ce7f6372a69fc6727f1b

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-7d2a2d0e4912bf7db6fb447b482c7534a364fc7d

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-131d8ce15bd6e21ae44efc72506b8f250ef01726

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-MjFhMTMwMTM2N2I3OWE1OTZlN2M4Mz imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-MjFhMTMwMTM2N2I3OWE1OTZlN2M4Mz imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-21a1301367b79a596e7c833f966297eeeb013334 imageSuffix=-dev

**Comment by CandelR** [comment]
/update-preview drive-server preview-443-21a1301367b79a596e7c833f966297eeeb013334 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-21a1301367b79a596e7c833f966297eeeb013334 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by CandelR** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by CandelR** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-40ccdc99e3c0dd474cde7783ee15c6554831b3ca

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-ae6775af62a9f0a12958f8653d57b7e6686c400b

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-3e8b311c9174a9cac4eb589238c74f93970daca1

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-e9962d40d31f4451e228bb7a3e14af148f827c9f

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-fefd5e4729a675234b7e86b8845b0029fa2da5a1

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by CandelR** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-869db915d42398580aa4b2415c85ebcdafe2019f

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-f2b11b096112f3a38ccce5f0e067fd3ae4ccfb66

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview payments preview-73-9e42accbbcb80bca704929b494858127831db0f3 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview payments preview-73-9e42accbbcb80bca704929b494858127831db0f3 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview payment preview-73-9e42accbbcb80bca704929b494858127831db0f3 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview payments preview-73-9e42accbbcb80bca704929b494858127831db0f3 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview payments-dev preview-73-9e42accbbcb80bca704929b494858127831db0f3

**Comment by apsantiso** [comment]
/update-preview payments-dp preview-73-9e42accbbcb80bca704929b494858127831db0f3 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview payments preview-73-9e42accbbcb80bca704929b494858127831db0f3 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview payments-server preview-73-9e42accbbcb80bca704929b494858127831db0f3 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by sg-gs** [comment]
Reviewed the description of this PR to reduce noise @apsantiso

**Comment by sg-gs** [comment]
Update the PR description again to mark new modules, models and repositories as merged. @apsantiso

**Comment by CandelR** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by inxtci** [comment]
> [!CAUTION]
> Preview failed: Not deployed after 5 minutes check 💥
> **Last status received: Progressing**

**Comment by inxtci** [comment]
> [!CAUTION]
> Pull request not updated after 5 minutes 💥
> We tried to update the image with drive-server-wip-dev:preview-287-7102a6571c661d338852994492206782dcdb57f8 
> If the arguments are correct, please check the logs. Or try again with another deployment, tag, or imageSuffix configuration.

**Comment by inxtci** [comment]
> [!CAUTION]
> Preview failed: Not deployed after 5 minutes check 💥
> **Last status received: Progressing**

**Comment by inxtci** [comment]
> [!CAUTION]
> Pull request not updated after 5 minutes 💥
> We tried to update the image with drive-server-wip-dev:preview-287-bf270ebe0e6ec03a12247338b58414e999cdc0e4 
> If the arguments are correct, please check the logs. Or try again with another deployment, tag, or imageSuffix configuration.

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-bf270ebe0e6ec03a12247338b58414e999cdc0e4

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview payments-server preview-75-a77a8869c6845177649d5f06dea17686a040402d imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview payments-server preview-73-f97aa0c8877a9268e56395246774fa9349a57d04 imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by inxtci** [comment]
> [!NOTE]
> Pull request successfully updated 🚀
> Preview updated with drive-server-wip-dev:preview-287-e85a282c97625d30d0440eabe41d7e21aaa3d8e4

**Comment by inxtci** [comment]
> [!CAUTION]
> Preview failed: Not deployed after 5 minutes check 💥
> **Last status received: Missing**

**Comment by apsantiso** [comment]
/update-preview drive-server preview-443-2d0b6c342eacc3b0604e161b53e5d2f80e8cfb5f imageSuffix=-dev

**Comment by sg-gs** [comment]
Environment variables added. Deploying.

**Pull request merged**